README.md

```diff
@@ -93,6 +93,70 @@ Possible values are:
 * `messages`: display requests and responses notifications
 * `verbose`: display each request and response as JSON
 
+## How does ruby-lsp differ from Solargraph?
+
+**Background**
+
+In July 2021, Shopify quietly released [rubocop-lsp](https://github.com/Shopify/rubocop-lsp). In March 2022, that was succeeded by [ruby-lsp](https://github.com/Shopify/ruby-lsp).
+As the name suggests, the newer library is intended to have a wider set of capabilities.
+It has a corresponding [VS Code extension](https://github.com/Shopify/vscode-ruby-lsp).
+ruby-lsp is widely used at Shopify, and is installed by default as part of most Ruby or Rails codebases.
+
+Solargraph is an older project, first released in March 2017.
+It was developed by [Castwide Technologies](https://castwide.com/), an Columbus, OH based company who provide various services including web hosting and software development.
+It also has a corresponding [VS Code extension](https://marketplace.visualstudio.com/items?itemName=castwide.solargraph).
+
+**Major Differences**
+
+The biggest difference between the two libraries is their approach to understanding the structure your code, and its dependencies.
+This is used to provide capabilities such as intellisense (context-aware auto-completion) and navigation (e.g. Go To Definition).
+
+Solargraph relies on [YARD](https://yardoc.org/), a popular tool for building documentation.
+Developers run `yard gems` command on their local machine to generate this information
+
+ruby-lsp does not provide direct support for these features.
+The intention is that they are handled by Sorbet’s LSP server, running alongside ruby-lsp.
+This means some functionality is only available if the files are [typed at ‘true’ or higher](https://sorbet.org/docs/static#file-level-granularity-strictness-levels).
+However there is some experimental work [in progress](https://github.com/Shopify/ruby-lsp/pull/429) to improve support for untyped code.
```

Comment:
Sorbet is not required to use the Ruby LSP, even though you can use both at the same time. I wouldn't mention this.